addons/hw_drivers/iot_object_info.py

```diff
@@ -0,0 +1,41 @@
+
+import logging
+from threading import Thread
+
+_logger = logging.getLogger(__name__)
+
+
+class IoTObjectInfo:
```

Comment:
Shouldn't that be called `Handler`?
Another idea (don't know if good or bad), we could make it inherit from `Thread`, to avoid making both `Driver` and `Interface` inherit from it?

```suggestion
class Handler(Thread):
```